homework_1/task_1/fizzbuzz/fizzbuzz.go

```diff
@@ -0,0 +1,32 @@
+package fizzbuzz
+
+import (
+	"strconv"
+
+	"github.com/pkg/errors"
+)
+
+func Fizzbuzz(start, end int) ([]string, error){
+	if end < start {
+		return nil, errors.New("The end flag should be greater that the start flag.")
```

Comment:
No punctuation in error messages!

https://axilis.slack.com/archives/C020JHMJ274/p1620815316002600?thread_ts=1620815275.002100&cid=C020JHMJ274